content/en/docs/installation/cli.md

```diff
@@ -4,72 +4,43 @@ description: Install the notation CLI on Linux, macOS, and Windows
-        ```
-    2. For **Linux** use [sha256sum](https://www.unix.com/man-page/linux/1/sha256sum) command to generate hash of the archive file.
-        ```shell
-        $ sha256sum notation_1.0.0-rc.1_darwin_arm64.tar.gz
-        eaa7b0c7c8d18e504766ce8d3ac5e46da2e97f4fdcead8be997e0ae74b146b00  notation_1.0.0-rc.1_darwin_arm64.tar.gz
-        ```
-    3. For **Window** use the [Get-FileHash](https://learn.microsoft.com/powershell/module/microsoft.powershell.utility/get-filehash?view=powershell-7.3) command in powershell to generate the hash of the archive file.
-        ```powershell
-        (Get-FileHash .\notation_1.0.0-rc.1_windows_amd64.zip).Hash
-        7d091cbd62886d1b47b60519a5b56314e794caf18751b1cccab2f54387a0d5c4
-        ```
-3. Compare the results of the command with the value shown in the `notation_<version>_checksums.txt` file. The two values should match.
-
-Alternatively, on macOS and linux you can also use `--check` mode to read the checksum file and verify the integrity of the archive.
-```shell
-$ shasum --check notation_1.0.0-rc.1_checksums.txt
-shasum: notation_1.0.0-rc.1_linux_arm64.tar.gz: No such file or directory
-notation_1.0.0-rc.1_linux_arm64.tar.gz: FAILED open or read
-shasum: notation_1.0.0-rc.1_darwin_amd64.tar.gz: No such file or directory
-notation_1.0.0-rc.1_darwin_amd64.tar.gz: FAILED open or read
-shasum: notation_1.0.0-rc.1_linux_amd64.tar.gz: No such file or directory
-notation_1.0.0-rc.1_linux_amd64.tar.gz: FAILED open or read
-shasum: notation_1.0.0-rc.1_windows_amd64.zip: No such file or directory
-notation_1.0.0-rc.1_windows_amd64.zip: FAILED open or read
-notation_1.0.0-rc.1_darwin_arm64.tar.gz: OK
-shasum: WARNING: 4 listed files could not be read
-```
 
-## Install the CLI for macOS
+## Download and install the CLI for Linux
 
-Expand the archive file directly into `/usr/local/bin`:
+Download the latest stable release of the notation CLI binary for macOS and checksum file, then verify the integrity of the download.
+
+For ARM processors:
 
 ```console
-tar xvzf <ARCHIVE_FILE> -C /usr/local/bin/ notation
+curl -LO https://github.com/notaryproject/notation/releases/download/v1.0.0-rc.2/notation_1.0.0-rc.2_linux_arm64.tar.gz
+curl -LO https://github.com/notaryproject/notation/releases/download/v1.0.0-rc.2/notation_1.0.0-rc.2_checksums.txt
+shasum --check notation_1.0.0-rc.2_checksums.txt
 ```
 
-Alternatively, you can expand the archive file to a different directory and add it to your path. For example, the following command expands the archive file to `<EXAMPLE_PATH>/notary-cli/` and adds it to your path using the `PATH` variable in `~/.zshhrc`.
+For x86 processors:
 
 ```console
-tar xvzf <ARCHIVE_FILE> -C <EXAMPLE_PATH>/notation-cli/ notation
-echo 'export PATH="$PATH:<EXAMPLE_PATH>/notary-cli/"' >> ~/.zshrc
+curl -LO https://github.com/notaryproject/notation/releases/download/v1.0.0-rc.2/notation_1.0.0-rc.2_linux_amd64.tar.gz
```

Comment:
Can we use an environment variable for the Notation version? This will keep minimum changes when the version is changed.

Reply:
Agreed.  fixed.